node/batcher/utils_test.go

```diff
@@ -108,12 +110,17 @@ func createBatchers(t *testing.T, num int, shardID types.ShardID, batcherNodes [
 
 		key, err := x509.MarshalPKCS8PrivateKey(batcherNodes[i].sk)
 		signer := crypto.ECDSASigner(*batcherNodes[i].sk)
+		require.NoError(t, err)
+
+		configStore, err := os.MkdirTemp("", fmt.Sprintf("config-store-batcher%d%d", i, shardID))
+		require.NoError(t, err)
```

Comment:
You have to clean it when the test ends... use `t.TempDir` and it will happen automatically